### ISSUE TITLE
Project roadmap & Youtube TFRS series schedule

### ISSUE THREAD

**Issue by ebstan** [tensorflow/recommenders#349]
Hi, can i just find out if there is any plan to release the project roadmap for this project? Separately, just wondering how long or how many video episodes are there in the pipeline for the TFRS series on Youtube? Would love to learn what are the future topics that will be covered in the series. Thank you!

**Comment by windmaple** [comment]
Yes, we will work on new videos on TFRS. Specific timelines/topics are not decided yet.

If there are things that you would like to learn, please add them here and we will consider covering them for sure :)
